### ISSUE TITLE
Yolov4 and Yolov4-Tiny

### ISSUE THREAD

**Issue by sanjel0307** [AlexeyAB/darknet#8447]
I was just wondering is anyone able to note the distinction between yolov4 and tiny in terms of network size. i know they both use CSPDarknet but i can't seem to find any information about the size and number of layers.

**Comment by stephanecharette** [comment]
See:  https://www.ccoderun.ca/darkmark/Configuration.html#ConfigNew

I also have some youtube videos where I compare the results, such as https://www.youtube.com/watch?v=gPP6fh8IIAo

**Comment by sanjel0307** [comment]
Interesting, it says that yolov4 has 162 layers, yet i thought it used CSP darknet as the back-end in which my thought only had 106 layers? Could you please help me understand my misunderstanding.

**Comment by DeemAQ** [comment]
Hi, did you find the answer? I also would like to know the details of network used in yolov4-custom.cfg 
so far the only thing I know that it does not use cspdarknet as backbone but i still haven't figured which version of darknet is it (darknet-53 maybe?)

**Comment by stephanecharette** [comment]
Did you look at the config files?  I believe there are 84 of them.  Should be obvious what uses CSP:

```
stephane@u2004-hankai ~/s/p/build (master)> dir ~/src/darknet/cfg/*.cfg | wc -l
84
stephane@u2004-hankai ~/s/p/build (master)> dir ~/src/darknet/cfg/ | grep csp
-rw-rw-r-- 1 stephane stephane  28K Jan  3 13:57 cspx-p7-mish.cfg
-rw-rw-r-- 1 stephane stephane  31K Jan  3 13:57 cspx-p7-mish_hp.cfg
-rw-rw-r-- 1 stephane stephane  15K Jan  3 13:57 cspx-p7-mish-omega.cfg
-rw-rw-r-- 1 stephane stephane  14K Jan  3 13:57 yolov4-csp.cfg
-rw-rw-r-- 1 stephane stephane 9.3K Jan  3 13:57 yolov4-csp-s-mish.cfg
-rw-rw-r-- 1 stephane stephane  14K Jan  3 13:57 yolov4-csp-swish.cfg
-rw-rw-r-- 1 stephane stephane  16K Jan  3 13:57 yolov4-csp-x-mish.cfg
-rw-rw-r-- 1 stephane stephane  16K Jan  3 13:57 yolov4-csp-x-swish.cfg
-rw-rw-r-- 1 stephane stephane  16K Jan  3 13:57 yolov4-csp-x-swish-frozen.cfg
-rw-rw-r-- 1 stephane stephane  15K Jan  3 13:57 yolov4-sam-mish-csp-reorg-bfm.cfg
```

And, as I posted above 3 weeks ago, see this link for a bit more info on some of these config files:  https://www.ccoderun.ca/darkmark/Configuration.html#ConfigNew

**Comment by DeemAQ** [comment]
> Did you look at the config files? I believe there are 84 of them. Should be obvious what uses CSP

yes ofc! but it is not clear what yolov4 specifically use as a backbone...

> And, as I posted above 3 weeks ago, see this link for a bit more info on some of these config files: https://www.ccoderun.ca/darkmark/Configuration.html#ConfigNew

I also saw the link it only states that yolov4 has 3 layers of yolo and the custom cfg is similar to it with minor differences
